tensai-api/src/main/java/dev/phomc/tensai/networking/PluginMessage.java

```diff
@@ -28,6 +28,7 @@
 
 public abstract class PluginMessage {
 	public static final String CHANNEL_VFX = "tensai:vfx";
+	public static final String CHANNEL_KEYBINDING = "tensai:keybinding";
```

Comment:
I think a channel for managing client features like HUD, key bindings or permissions is better than a dedicated channel for key bindings.

Reply:
Dedicated channel for listening C2S i suppose

Reply:
The channel supports duplex communication: it can be both C2S or S2C at the same time.

Reply:
isn't it?